integration_tests/e2e/futureVisits.cy.ts

```diff
@@ -30,5 +35,18 @@ context('Bookings home page', () => {
     bookingsPage.visitStartTime(1).contains('10am')
     bookingsPage.visitEndTime(1).contains('11:30am')
     bookingsPage.visitReference(1).contains('ab-cd-ef-gh')
+
+    bookingsPage.visitLink(1).click()
```

Comment:
GH won't let me comment on line 22, but now that the test has been extended, perhaps update the description. E.g.:
```
it('should show bookings home page with a future visit and navigate to view the visit details', () => {
```
... or similar 🤷‍♂️